Lesson_11/Tusk_1.py

```diff
@@ -0,0 +1,20 @@
+from time import ctime
+
+
+def func_decorator(func):
+    def inside_func():
```

Comment:
#Use *args and **kwargs here to make in works with functions that take arguments. And pass these arguments to the func() call

Reply:
done, thank you